sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/reflect/DoFnSignature.java

```diff
@@ -280,17 +329,51 @@ public ResultT dispatch(PipelineOptionsParameter p) {
           new AutoValue_DoFnSignature_Parameter_ProcessContextParameter();
     private static final OnTimerContextParameter ON_TIMER_CONTEXT_PARAMETER =
         new AutoValue_DoFnSignature_Parameter_OnTimerContextParameter();
+    private static final TimestampParameter TIMESTAMP_PARAMETER =
```

Comment:
FWIW you could easily break out a boilerplate PR. Combined with doing less autoformatting it would significantly speed up and improve the quality on review here.